include/InstrumentMidiIOView.h

```diff
@@ -72,7 +72,7 @@ class InstrumentMiscView : public QWidget
 	InstrumentMiscView( InstrumentTrack *it, QWidget* parent );
 	~InstrumentMiscView();
 
-	GroupBox * getPitchGroupBox()
+	GroupBox * PitchGroupBox()
```

Comment:
@serdnab This is [our convention](https://github.com/LMMS/lmms/wiki/Coding-conventions), too.